File: Server-C#-EntityFramework/WebApplication1/Models/Page.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace WebApplication1.Models
{
	[Table("page")]
	public class Page
	{
		[Column("id")]
		public int Id { get; set; }
		[Column("user_id")]
		[JsonPropertyName("user_id")]
		public required int UserId { get; set; }
		[Column("url")]
		public required string Url { get; set; }
		[Column("title")]
		public required string Title { get; set; }
		[Column("content")]
		public string? Content { get; set; }
		[Column("date_published")]
		[JsonPropertyName("date_published")]
		public long? DatePublished { get; set; }
		[Column("date_modified")]
		[JsonPropertyName("date_modified")]
		public long? DateModified { get; set; }
	}
}

File: Client-Dart-Flutter/simple_blog_flutter_application/lib/Models/page.dart
class Page {
  int ?id;
  int userId;
  String title;
  String url;
  String ?content;
  int ?datePublished;
  int ?dateModified;

  Page({
    this.id, 
    required this.userId, 
    required this.title, 
    required this.url, 
    this.content, 
    this.datePublished, 
    this.dateModified
  });

  factory Page.fromJson(Map<String, dynamic> json) {
    return Page(
      id: json['id'],
      userId: json['user_id'],
      title: json['title'],
      url: json['url'],
      content: json['content'],
      datePublished: json['date_published'],
      dateModified: json['date_modified'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'user_id': userId,
      'title': title,
      'url': url,
      'content': content,
      'date_published': datePublished,
      'date_modified': dateModified,
    };
  }
}